File: typescript-sdk/src/examples/fi_internal_to_card.ts
import { PayoutBuilder } from '../dx/builder';

async function main() {
	const builder = PayoutBuilder.create();
	const res = await builder
		.forOriginator('fi-001')
		.withFundingInternal(true, 'conf-123')
		.toCardDirect('tok_pan_411111******1111')
		.forAmount('USD', 101) // odd -> executed
		.withIdempotencyKey('demo-internal-card-1')
		.execute();
	console.log('Payout result:', { status: res.status, payoutId: res.payoutId });
}

main().catch(err => { console.error(err); process.exit(1); });

File: python-sdk/examples/fi_internal_to_card.py
from visa_direct_sdk.dx.builder import PayoutBuilder


def main():
	builder = PayoutBuilder.create()
	res = builder \
		.for_originator("fi-001") \
		.with_funding_internal(True, "conf-123") \
		.to_card_direct("tok_pan_411111******1111") \
		.for_amount("USD", 101) \
		.with_idempotency_key("py-demo-internal-card-1") \
		.execute()
	print({"status": res.get("status"), "payoutId": res.get("payoutId")})


if __name__ == "__main__":
	main()
